src/actions.rs

```diff
@@ -323,6 +323,7 @@ pub fn steps_system(
 */
 #[derive(Debug)]
 pub struct ConcurrentlyBuilder {
+    mode: ConcurrentMode,
```

Comment:
I think my preference here would be to add a new Action instead, called `Race`, which is what this kind of operation is usually called, ime, as opposed to parameterizing `Concurrently`. Although if we're gonna go that way, maybe Concurrently should be renamed to `Join`, too...

Reply:
ah interesting. Where do you get those names from? Do you have a resource about utilityAI?

Reply:
this naming is more from the world of threading/async :)

I like splitting them up even with that duplication because it's nicer to have individual but very simple building blocks, rather than increasingly parameterizing the same thing to tbh completely change their semantics.

Reply:
I split up between `Race` and `Join`, removed `Concurrently` from the prelude, and added a cool little example that show clearly how `Race` and `Join` work :)